prac_06/languages.py

```diff
@@ -0,0 +1,13 @@
+from prac_06.programming_language import ProgrammingLanguage
+
+
+def main():
+    python = ProgrammingLanguage("Python", "Dynamic", True, 1991)
+    ruby = ProgrammingLanguage("Ruby", "Dynamic", True, 1995)
+    visual_basic = ProgrammingLanguage("Visual Basic", "Static", False, 1991)
+    print(python)
+    print(ruby)
+    print(visual_basic)
+
```

Comment:
The dynamic programming languages must be displayed in the output for languages program. This code u have written is part of the programming languages code.